Dockerfile

```diff
@@ -10,18 +10,22 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 
-# critical package deps preventing update: openjdk-7-jdk, libnspr4-0d, libwxgtk2.8-0, libicu52
-FROM ubuntu:14.04
-
+FROM buildpack-deps:jessie as ntr-base
 ENV MAVEN_VERSION 3.5.2
-ENV DEBIAN_FRONTEND noninteractive
 ENV MAVEN_HOME /usr/share/maven
+# install maven
+RUN curl -fsSL http://archive.apache.org/dist/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz | tar xzf - -C /usr/share \
+  && mv "/usr/share/apache-maven-${MAVEN_VERSION}" /usr/share/maven
 
-RUN groupadd -r couchdb && useradd -d /couchdb -g couchdb couchdb
+# lean node setup to be re-used later
+FROM ntr-base as ntr-node
```

Comment:
seems to be not used for now, could we comment this out until the nodejs image is used?